hack/crd-overlay.yml

```diff
@@ -8,6 +8,16 @@ metadata:
   #@overlay/remove
   #@overlay/match missing_ok=True
   annotations:
+spec:
+  versions:
+  #@overlay/match by=overlay.index(0)
```

Comment:
Open to thoughts here on if there is a better way to match. 

We need the overlay since it seems `description` for metadata causes issues: https://github.com/kubernetes-sigs/controller-tools/pull/511.

Reply:
should we do `overlay.all,expects="0+"` since this would be true for all versions.